backend/dynamic-ui/http-data-modelling.ftd

```diff
@@ -0,0 +1,255 @@
+-- import: fifthtry.github.io/admonitions as cbox
+
+-- ds.page: Basics of `http` processor and `data modelling`
+
+In this part, we will delve into essential concepts like the `http` processor
+and `data modelling` through a concise yet illuminating example.
+
+To facilitate your learning journey, we will take a pratical scenario involving 
+an array of records. Each record has two key attributes: the **country-name** 
+and it's corresponding **capital**, in the form of JSON data.
```

Comment:
```suggestion
We will  an array of records. Each record has two key attributes: the **country-name** 
and it's corresponding **capital**, in the form of JSON data.
```

Use simpler language. Cut words that are not needed. Avoid adjectives if you can.